aten/src/ATen/cpu/vec256/vec256_int.h

```diff
@@ -95,25 +95,37 @@ struct Vec256<int64_t> : public Vec256i {
     return _mm256_cmpeq_epi64(values, other.values);
   }
   Vec256<int64_t> operator!=(const Vec256<int64_t>& other) const {
-    auto zero = _mm256_set1_epi64x(0);
```

Comment:
just replace this line with `auto zero = _mm256_set1_epi64x(-1);`, it would be cheaper than the new alternative